### PR TITLE
Timeout cnw

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import com.wikia.webdriver.common.core.api.WikiFactory;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.dataprovider.CreateNewWikiDataProvider;
+import com.wikia.webdriver.common.logging.Log;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.logging.Log'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java [hunk 2]
```diff
@@ -586,7 +586,7 @@ public void verifyTOCsectionLinkWorks(int numberOfTheSection) {
   }
 
   public void verifyWikiTitleOnCongratualtionsLightBox(String wikiName) {
-    wait.forElementVisible(welcomeLightBoxTitle);
+    wait.forElementVisible(welcomeLightBoxTitle, 30);
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java]: Can we move `30` to the `const` with some explicit name? Or at least leave the comment here why `30`

File: src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java [hunk 3]
```diff
@@ -34,6 +34,7 @@ public void createNewWikiLangTC001(String lang) {
     cnw2.selectCategory(CreateWikiMessages.WIKI_CATEGORY_ID);
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java]: This whole tests is 💩 . At least these name `CreateNewWikiPageObjectStepN` could be improved...

**Pull request merged**